tests/test_transpile.py

```diff
@@ -256,6 +256,14 @@ def test_if(self):
         )
         self.validate("SELECT IF(a > 1, 1) FROM foo", "SELECT CASE WHEN a > 1 THEN 1 END FROM foo")
 
+    def test_if_null(self):
```

Comment:
i think we should transpile all IFNULL calls to COALESCE in every dialect, so let's validate that.

delete this test and write a new one in test_dialects where you can use the validate_all command

remove the IFNULL mapping that you added to bigquery, and also remove the IFNULL in Tableau.

add this mapping to the base .py FUNCTIONS